service-registry/src/test/java/org/apache/servicecomb/serviceregistry/client/http/TestClientPool.java

```diff
@@ -17,43 +17,16 @@
 
 package org.apache.servicecomb.serviceregistry.client.http;
 
-import org.apache.servicecomb.serviceregistry.config.ServiceRegistryConfig;
 import org.junit.Test;
 
-import io.vertx.core.http.HttpVersion;
-import mockit.Mock;
-import mockit.MockUp;
-
 public class TestClientPool {
   @Test
   public void testHttpClientPool() {
-    new MockUp<ServiceRegistryConfig>() {
-      @Mock
-      public HttpVersion getHttpVersion() {
-        return HttpVersion.HTTP_2;
-      }
-
-      @Mock
-      public boolean isSsl() {
-        return true;
-      }
-    };
     HttpClientPool.INSTANCE.create();
```

Comment:
There is no verification code here.

Reply:
oh, yes.
i only confirmed with author and deleted useless code, did not realize this is not a effective test case
and the next case have the same problem.

i will confirmed with author again.

Reply:
done
add new test cases for it.